src/bootstrap/test.rs

```diff
@@ -430,6 +430,10 @@ impl Step for Rustfmt {
             &[],
         );
 
+        if builder.config.cmd.bless() {
+            cargo.env("BLESS", "1");
```

Comment:
@oli-obk - This seems reasonable to me, though I'd like to change the var name to be a little more explicit (perhaps something like `RUSTC_BLESS`?) so that it's somewhat more explicit especially when working with rustfmt out of tree.

Wanted to make sure you didn't have any objections or alternative naming suggestions before I opened a PR?

Reply:
This is currently being worked on in https://github.com/rust-lang/rust/pull/113298